roles/handy/app/tasks/main.yml

```diff
@@ -17,3 +17,9 @@
 - name: Configure Eye
   action: copy src=handy.eye.rb dest=/u/eye/handy.eye
   notify: Reload Eye config
+
+- name: Install Dependencies
+  action: apt package={{item}} state=installed
+  with_items:
+    - wkhtmltopdf
+  notify: Installing dependencies
```

Comment:
In Ansible, a role has a special set of tasks called "handlers" that are only performed when triggered by a normal task using `notify`. The task above notifies the "Reload Eye config" handler here: https://github.com/Pitt-CSC/ansible/blob/master/roles/handy/app/handlers/main.yml. This task doesn't need to notify any handlers.